src/Models/BaseElement.php

```diff
@@ -13,80 +12,62 @@
 use SilverStripe\DataObjectPreview\Controllers\DataObjectPreviewController;
 use SilverStripe\Forms\CheckboxField;
 use SilverStripe\Forms\DropdownField;
-use SilverStripe\Forms\GridField\GridFieldAddExistingAutocompleter;
-use SilverStripe\Forms\GridField\GridFieldAddNewButton;
-use SilverStripe\Forms\GridField\GridFieldConfig_Base;
-use SilverStripe\Forms\GridField\GridFieldDataColumns;
-use SilverStripe\Forms\GridField\GridFieldDeleteAction;
-use SilverStripe\Forms\GridField\GridFieldDetailForm;
 use SilverStripe\Forms\HiddenField;
-use SilverStripe\Forms\LiteralField;
 use SilverStripe\Forms\ReadonlyField;
 use SilverStripe\Forms\TextField;
 use SilverStripe\ORM\ArrayList;
 use SilverStripe\ORM\CMSPreviewable;
 use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\DB;
 use SilverStripe\ORM\FieldType\DBField;
-use SilverStripe\ORM\FieldType\DBHTMLText;
 use SilverStripe\ORM\Search\SearchContext;
 use SilverStripe\Security\Permission;
 use SilverStripe\SiteConfig\SiteConfig;
 use SilverStripe\Versioned\Versioned;
 use SilverStripe\View\Parsers\URLSegmentFilter;
 use SilverStripe\Core\Injector\Injector;
+use SilverStripe\View\SSViewer;
 
-/**
- * @package elemental
- */
 class BaseElement extends DataObject implements CMSPreviewable
 {
-
     /**
      * Override this on your custom elements to specify a cms icon
      * @var string
      */
     private static $icon = 'elemental/images/base.svg';
+
     /**
      * @var array $db
      */
-    private static $db = array(
+    private static $db = [
         'Title' => 'Varchar(255)',
         'Sort' => 'Int',
-        'Enabled' => 'Int',
-        'ExtraClass' => 'Varchar(255)',
-        'AvailableGlobally' => 'Boolean(1)'
-    );
+        'ExtraClass' => 'Varchar(255)'
+    ];
 
     /**
      * @var array $has_one
      */
-    private static $has_one = array(
-        'Parent' => ElementalArea::class,
-        'List' => ElementList::class // optional.
-    );
+    private static $has_one = [
+        'Parent' => ElementalArea::class
+    ];
 
     /**
-     * @var array $has_many
+     * @var array $extensions
```

Comment:
We don't actually need `$extensions` here as it's a description for the `@var` construct - it'd be better to remove it and the others in this class